docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,123 @@ create_base_gui(appdata_s *ad)
    }
```

Comment:
Line 506:
Kindly rephrase the line as "1. To use the App Control API (in mobile and wearable), application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:"

Line 521:
Kindly rephrase as "Launch Applications Using Applications Data"

Line 533:
Kindly rephrase as "When an `app_control` instance is created, you must set the operation and the MIME type. For example, the app_control launches an application that is set with the `APP_CONTROL_OPERATION_VIEW` operation and the image/jpeg MIME type. The operation is a mandatory information for the launch request. If the operation...."

Line 547:
Kindly rephrase as "For more information on the common application services, see [Common Application Controls](common-appcontrol-n.md)."

Line 549:
Kindly rephrase as "3. Add additional data to the app_control instance using `app_control_add_extra_data()` or `app_control_add_extra_data_array()`. In this example, a message is added as an additional data:"

Line 557:
Kindly rephrase the sentence by applying the Global Comment.

Line 567:
Kindly rephrase as "Since Tizen 2.4, service applications are allowed to launch explicitly and the caller must be an application in the same package. Otherwise, the `app_control` instance returns an error."

Line 569:
Kindly rephrase as "In the launched application, `app_control` instance use `app_control_foreach_extra_data()` to read the additional data set…. "

Line 591:
Kindly rephrase as "After completing all the tasks of application control, you need to clean it. To perform clean up pass the `app_control` handle to `app_control_destory()`."

Line 600:
Kindly rephrase as "Control Launch Mode"

Line 647:
Kindly rephrase as "Use `app_control_set_launch_mode()` to define the launch mode for the application. The second parameter defines the launch mode with `app_control_launch_mode_e` enumerator..."

Reply:
Done